doc/basics/overlay_tutorial_5.py

```diff
@@ -1,27 +1,28 @@
 import os
 from asyncio import ensure_future, get_event_loop
+from dataclasses import dataclass
 
 from pyipv8.ipv8.community import Community
 from pyipv8.ipv8.configuration import ConfigBuilder, Strategy, WalkerDefinition, default_bootstrap_defs
 from pyipv8.ipv8.lazy_community import lazy_wrapper
-from pyipv8.ipv8.messaging.lazy_payload import VariablePayload, vp_compile
+from pyipv8.ipv8.messaging.payload_dataclass import overwrite_dataclass
 from pyipv8.ipv8_service import IPv8
 
+dataclass = overwrite_dataclass(dataclass)
```

Comment:
**Question:** I had to lookup why this overwrite was necessary. Since this file is part of a tutorial, is it clear for the reader why this logic is here? If not, I would recommend to add a comment.

Reply:
Thanks for the suggestion. I'll add a comment to this line.